modules/nodes-pods-user-namespaces-configuring.adoc

```diff
@@ -35,7 +35,7 @@ Enabling the `TechPreviewNoUpgrade` feature set on your cluster cannot be undone
 +
 After you save the changes, new machine configs are created, the machine config pools are updated, and scheduling on each node is disabled while the change is being applied.
 
-* You enabled the crun container runtime on the worker nodes. crun is currently the only released OCI runtime with support for user namespaces.
+* The crun container runtime is running on the worker nodes. crun is currently the only released OCI runtime with support for user namespaces. crun is active by default.
```

Comment:
IIRC, `runc` also supports user namespaces.

https://frasertweedale.github.io/blog-redhat/posts/2021-05-27-oci-runtime-spec-runc.html

Reply:
1.2.0 does but we haven't packaged it yet

Reply:
Changed the sentence to:

> crun is currently the only OCI runtime packaged with {product-title} that supports user namespaces.